rs/compression/src/noc/noc.rs

```diff
@@ -0,0 +1,48 @@
+use std::fs::File;
+use std::io::{BufWriter, Write};
+
+use anyhow::Result;
+use utils::io::wrap_write;
+
+use crate::compression::IntSeqEncoder;
+
+pub struct NoEncoder {
+    size: usize,
+    sequence: Vec<u64>,
+}
+
+impl NoEncoder {
+    pub fn new(size: usize) -> Self {
+        Self {
+            size,
+            sequence: Vec::new(),
+        }
+    }
+}
+
+impl IntSeqEncoder for NoEncoder {
+    fn new_encoder(_universe: Option<usize>, size: usize) -> Self {
+        Self::new(size)
+    }
```

Comment:
remove

Reply:
actually we need this because IvfWriter will need a generic way to create an encoder, which is a templatized parameter to the struct.